src/main/java/frc/robot/Robot.java

```diff
@@ -101,55 +103,113 @@ public void robotInit() {
     // Controller bindings here
     controller.start().onTrue(Commands.runOnce(() -> swerve.setYaw(Rotation2d.fromDegrees(0))));
 
+    controller.x().toggleOnTrue(backAndForth());
+
     // Test binding for autoaim
     controller
         .a()
         .whileTrue(
-            swerve.pointTowardsTranslation(
+            swerve.pointTowardsTranslationCmd(
                 () -> -controller.getLeftY() * SwerveSubsystem.MAX_LINEAR_SPEED,
                 () -> -controller.getLeftX() * SwerveSubsystem.MAX_LINEAR_SPEED));
 
     NamedCommands.registerCommand("stop", swerve.stopWithXCmd().asProxy());
 
-    controller.b().toggleOnTrue(autoAimDemo(() -> swerve.getVelocity()));
+    controller
+        .b()
+        .whileTrue(
+            autoAimDemo(
+                () -> {
+                  swerve.polarDistance =
+                      Math.sqrt(
+                          Math.pow(swerve.getVelocity().vxMetersPerSecond, 2)
+                              + Math.pow(swerve.getVelocity().vyMetersPerSecond, 2));
+                  swerve.polarRadians =
+                      Math.atan2(
+                          swerve.getVelocity().vyMetersPerSecond,
+                          swerve.getVelocity().vxMetersPerSecond);
+
+                  swerve.polarDistance =
+                      MathUtil.clamp(
+                          swerve.polarDistance,
+                          -SwerveSubsystem.MAX_LINEAR_SPEED / 2,
+                          SwerveSubsystem.MAX_LINEAR_SPEED / 2);
+                  Logger.recordOutput(
+                      "AutoAim/Polar Sppeeds",
+                      new ChassisSpeeds(
+                          swerve.polarDistance * Math.cos(swerve.polarRadians),
+                          swerve.polarDistance * Math.sin(swerve.polarRadians),
+                          swerve.getVelocity().omegaRadiansPerSecond));
+                  return new ChassisSpeeds(
+                      swerve.polarDistance * Math.cos(swerve.polarRadians),
+                      swerve.polarDistance * Math.sin(swerve.polarRadians),
+                      swerve.getVelocity().omegaRadiansPerSecond);
+                }));
   }
 
   @Override
   public void robotPeriodic() {
     CommandScheduler.getInstance().run();
   }
 
+  public Command backAndForth() {
```

Comment:
if you do keep this, drop one of the sequences and call `.repeatedly()` on the remaining sequence so it goes back and forth repeatedly